lib/utils/transaction.js

```diff
@@ -109,7 +109,8 @@ function initData(tx, data) {
       const keys = Object.keys(data);
       tx._fields.forEach(function(field) {
         if (keys.indexOf(field) !== -1) {
-          self[field] = data[field];
+          const val = data[field];
+          self[field] = val === "" ? "0x" : val;
```

Comment:
I think if `val` is a string that isn't prefixed with 0x we should probably just prefix it here? Just to be safe?

Reply:
:+1: